1/Dockerfile.rhel7

```diff
@@ -46,7 +46,9 @@ COPY ./contrib/openshift /opt/openshift
 COPY ./contrib/jenkins /usr/local/bin
 ADD ./contrib/s2i /usr/libexec/s2i
 
-RUN /usr/local/bin/fix-permissions /opt/openshift && \
+RUN mkdir -p /tmp/openshift/plugins && \
+    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.hpi && \
```

Comment:
test doesn't seem to like this.  does the plugins directory exist at the point at which you are running this command in the docker build?